packages/jest-config/src/normalize.ts

```diff
@@ -1068,14 +1067,9 @@ export default async function normalize(
   if (!newOptions.watchAll) {
     newOptions.watchAll = false;
   }
-
-  // as unknown since it can happen. We really need to fix the types here
-  if (
-    newOptions.moduleNameMapper === (DEFAULT_CONFIG.moduleNameMapper as unknown)
-  ) {
+  if (!newOptions.moduleNameMapper) {
     newOptions.moduleNameMapper = [];
```

Comment:
also removed this from the default because its type causes conflict.
we are setting the normalized default value here in case people don't pass any value